### DISCUSSION TITLE
Prodotti SonOff instabili

### DISCUSSION THREAD

**Discussion by Virgilio123** [indomus/forum#1014] [GENERALE - Discussioni generiche]
Buongiorno a tutti,

ho acquistato due sonoff mini r2 e un sonoff  basic.
ho installato il firmaware tasmota_lite versione 10.0 saldando direttamente i i fili sul circuito stampato senza alcun problema.
ho configurato correttamente la connessione wi-fi e i parametri mqqt.

Tutto ha funzionato per alcuni giorni, ma ora un sonoff mini si blocca continuamente (funziona due minuti e poi va off line), mentre il sonoff basic non riesce più ad agganciare la rete wifi (non è cambiato nulla).

Prima di domotizzare tutte le luci di casa volevo fare alcune prove, ma sono rimasto sconcertato dalla instabilità dei SonOff !!!!
Non capisco se sia un problema di qualità dei sonoff (sono comunque dei prodotti cinesi) o un errore mio nella configurazione.

Forse la versione 10.0 di Tasmota ha qualche problema? Mi potete consigliare una versione sicuramente stabile?
in alternativa ai SonOff qualcuno ha provato altri prodotti che magari sono più stabili nel tempo?

**Comment by essegi65** [comment]
Io uso i sonoff sin dall'inizio e in effetti dove il segnale wifi arriva debolmente si possono riscontrare dei problemi di connessione, inoltre i sonoff sono dei prodotti più economici, p.e. rispetto a gli shelly, ma sono anche di qualità inferiore.
Comunque dove il segnale è stabile tutt'ora ne ho in funzione circa 8 (misti tra basic, pow, mini, ecc.) e con il FW 9.5 funzionano tranquillamente.

**Reply by Virgilio123** [reply]
Sicuramente non è un problema di segnale wifi, ho il router/wifi a pochi metri.
è decisamente frustrante avere un prodotto che funziona quando gli pare.
Comunque provo a installare la versione 9.5
Grazie

**Reply by essegi65** [reply]
Può capitare che qualche sonoff sia "fasullo" a me è successo su 2 basic che una volta flashati dopo poco tempo di funzionamento altalenante li ho dovuti buttare, ma gli altri per adesso stanno continuando a funzionare e fanno il suo (specialmente i pow che misurano anche la corrente). non smetterei di provare solo perchè alcuni ti danno problemi. Verifica anche il chip che montano perchè gli aggiornamenti recenti di tasmota non ricordo fino a che chip possono essere installati

**Reply by Virgilio123** [reply]
Il problema ora non c'è perchè sto facendo delle prove con i sonoff collegati alla presa elettrica, ma in un futuro quando sono installati dentro una cassetta di derivazione?
come faccio ad affidare l'accensione/spegnimento delle luci a questi aggeggetti che senza alcun motivo all'improvviso non funzionano più ?

Ho installato la versione di Tasmosta 10.0 su due sonoff mini, hanno iniziato a funzionare correttamente tutti e due, ora uno non va più!!!
forse sono meglio gli shelly 1 pro.

**Reply by essegi65** [reply]
Ti ripeto che anche io sto usando sonoff mini (attenzione i mini hanno bisogno di un condensatore per i disturbi elettrici trovi delle guide a tal proposito) e basic per luci led e funzionano. Certo che gli shelly sono migliori (ma ti bastano anche le versioni normali al posto dei PRO) e essendo migliori però hanno anche un costo differente

**Reply by Newreader** [reply]
Prova a reinstallare via OTA prima tasmota-minimal.bin.gz e successivamente tasmota-IT.bin.gz, ovviamente della versione 10.0. A me personalmente non hanno mai dato i problemi che tu hai evidenziato eppure ne ho un certo numero tra basic, mini, POW, TH10 come d'altronde ho anche gli shelly.

**Reply by Scialla** [reply]
Mi sa che hai qualche problema a casa tua, Tasmota 10 funziona benissimo. Prova a cambiare canale alla tua WiFi

**Reply by Virgilio123** [reply]
il sonoff incriminato, dopo essere stato scollegato e buttato in un cassetto tutta la notte oggi ricollegato alla rete funziona.

Ora volevo testare il prodotto prima di domotizzare tutta casa, ma se sono cosi "farlocchi" sinceramente devo cercare un prodotto diverso, anche se più costoso.
Non posso sicuramente ogni giorno sperare che tutto funzioni bene.

**Reply by Scialla** [reply]
Guarda che se danno problemi solo a te, forse dipende dalla situazione che hai in casa. Io ne ho parecchi e mai un problema in anni... indagherei sulla rete piuttosto

**Reply by Virgilio123** [reply]
la differenza di prezzo tra un sonoff e un prodotti di marche più blasonate è abissale, questo mi attirava molto, ma se su tre prodotti due funzionano a singhiozzo capisco la differenza di prezzo.

La rete non centra nulla, i sonoff sono tutti insieme sopra un tavolo, con il modem router a 30 centimetri.
Contento per Te che ti funzionano tutti da anni, ma credo che sia solo un caso fortunato.

**Reply by Scialla** [reply]
se vieni sul gruppo e chiedi, non sono un caso fortunato, forse il tuo è un caso sfortunato ma, per quel che costa, qualche prova a cambiare canale o a spostarli da eventuali causa di interferenze io la farei....
